LanguageFeatures/Control-flow-collections/Null-aware-elements/syntax_unary_prefix_A01_t01.dart

```diff
@@ -46,37 +44,74 @@ main() {
 
   var list = [
     ?++e1,
+//  ^
+// [analyzer] unspecified
```

Comment:
Isn't it a bug to report an error here? `e1 = e1 + 1` would have the type `int?`, which wouldn't (shouldn't) give rise to an `INVALID_NULL_AWARE_OPERATOR` warning.

Reply:
Reverted the change in this test and added a note to https://dart-review.googlesource.com/c/sdk/+/391621

Reply:
OK, thanks!